web3_proxy/src/bin/web3_proxy_cli/proxyd.rs

```diff
@@ -143,70 +154,160 @@ async fn run(
         }
     }
 
+    // start thread for watching config
+    // if let Some(top_config_path) = top_config_path {
+    //     let config_sender = spawned_app.new_top_config_sender;
+    //     {
+    //         thread::spawn(move || loop {
+    //             match fs::read_to_string(&top_config_path) {
+    //                 Ok(new_top_config) => match toml::from_str(&new_top_config) {
+    //                     Ok(new_top_config) => {
+    //                         if new_top_config != top_config {
+    //                             top_config = new_top_config;
+    //                             config_sender.send(top_config.clone()).unwrap();
+    //                         }
+    //                     }
+    //                     Err(err) => {
+    //                         // TODO: panic?
+    //                         error!("Unable to parse config! {:#?}", err);
+    //                     }
+    //                 },
+    //                 Err(err) => {
+    //                     // TODO: panic?
+    //                     error!("Unable to read config! {:#?}", err);
+    //                 }
+    //             }
+    //
+    //             thread::sleep(Duration::from_secs(10));
+    //         });
+    //     }
+    // }
```

Comment:
why is this commented out?

Reply:
Just checking, this is not a duplicate with this, right? https://github.com/yenicelik/web3-proxy/blob/stats_v3/web3_proxy/src/bin/web3_proxy_cli/proxyd.rs#L83 If so, I will remove this

Reply:
you are correct. this is a duplicate. delete this commented code block